test/common/child_process.js

```diff
@@ -119,7 +119,7 @@ function spawnSyncAndExit(...args) {
 }
 
 function spawnSyncAndExitWithoutError(...args) {
-  const spawnArgs = args.slice(0, args.length);
+  const spawnArgs = args.slice(0, args.length - 1);
```

Comment:
```suggestion
  const spawnArgs = args.slice(0, Math.max(args.length - 1, 2));
```
this will avoid the need to pass `undefined`

Reply:
`spawnSync` accepts more than 2 arguments though. I think I'm going to split it in two different utils, that should clear up the confusion and avoid those additional `undefined` arguments.